R/ScpModel-Class.R

```diff
@@ -279,16 +280,21 @@ scpModelFitElement <- function(object, name, what, filtered,
 }
 
 scpModelN <- function(object, name, filtered = TRUE) {
-    out <- scpModelFitElement(
-        object, name, "N", filtered, .runWorkflowMessage
-    )
+    assay_matrix <- scpModelInput(object, name, filtered)
```

Comment:
Please, don't use snake_case, but camelCase, so not assay_matrix but assayMatrix